vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -489,6 +489,11 @@ void ReactInstanceWin::InitJSMessageThread() noexcept {
 
   // Create MessageQueueThread for the DispatchQueue
   VerifyElseCrashSz(jsDispatchQueue, "m_jsDispatchQueue must not be null");
+
+  auto jsDispatcher =
+      winrt::make<winrt::Microsoft::ReactNative::implementation::ReactDispatcher>(Mso::Copy(jsDispatchQueue));
+  m_options.Properties.Set(ReactDispatcherHelper::JSDispatcherProperty(), jsDispatcher);
```

Comment:
> m_options.Properties.Set [](start = 1, length = 25)

I think it maybe right that we set it here, but  it will override the one set by users in the ReactInstanceSettings.
I would suggest removing the ReactInstanceSettings::JSDispatcher property.